### PR TITLE
adding go-cli

### PR DESCRIPTION
**Pull request by converge**
Adding `kubefirst cli` installer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,3 +126,75 @@ With that context in mind, once you've removed the manual things you may have ad
 ```
 docker run -it --env-file=kubefirst.env -v $PWD/gitops:/gitops -v $PWD/metaphor:/metaphor -v $PWD/scripts:/scripts -v $PWD/git:/git --entrypoint /scripts/nebulous/destroy.sh  nebulous:foo  
 ```
+
```
Comment (converge) [inline README.md:129]: what do you guys think to remove the previous README content? and continue working on the latest changes that are coming from GitLab?

**Review by converge — COMMENTED** [review]
it would be nice to remove unnecessary/auto generated comments

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,134 @@
+module gitlab.kubefirst.io/kubefirst/flare
+
+go 1.17
```
Comment (6za) [inline go.mod:3]: We need to rename this to point to proper repo.

File: main.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
+/*
+Copyright © 2022 Kubefirst Inc. devops@kubefirst.com
+
+*/
+package main
+
+import (
+	"gitlab.kubefirst.io/kubefirst/flare/cmd"
```
Comment (6za) [inline main.go:8]: Review local imports to new name of the repo.

File: terraform/main.tf [hunk 4]
```diff
@@ -0,0 +1 @@
+resource "null_resource" "foo" {}
```
Comment (6za) [inline terraform/main.tf:1]: Do we need this file for `create` ?

**Review by 6za — COMMENTED** [review]
Some points reviewing PR:

File: terraform/main.tf (hunk 4 above)
Comment (jarededwards) [reply terraform/main.tf:1]: no, feel free to delete the `terraform` directory, its from testing

**Comment by jarededwards** [comment]
> it would be nice to remove unnecessary/auto generated comments

I'm all for removing extra / auto generated code

File: README.md (hunk 1 above)
Comment (jarededwards) [reply README.md:129]: we might want to keep a link to our docs site but otherwise the content should be mostly out dated

**Review by jarededwards — DISMISSED** [review]

**Review by jarededwards — APPROVED** [review]

**Pull request merged**